daemon/endpoint.go

```diff
@@ -276,6 +276,8 @@ func (d *Daemon) createEndpoint(ctx context.Context, epTemplate *models.Endpoint
 		}
 	}
 
+	// The following docs describe the cases where the init identity is used:
+	// http://docs.cilium.io/en/latest/policy/lifecycle/#init-identity
```

Comment:
I expect that this URL will work when we merge the code (tested with the `#init-identity` added at the end locally and compared with the heading links online). If not I can follow up to make sure it's right.